Xwt.Gtk/Xwt.GtkBackend/AccessibleBackend.cs

```diff
@@ -157,6 +157,18 @@ public void InitializeBackend (object frontend, ApplicationContext context)
 			}
 		}
 
+		public virtual void AddChild (object nativeChild)
+		{
+			// TODO
+			throw new NotImplementedException ();
```

Comment:
It's better to do nothing here, otherwise we'll need to catch this exception everywhere. Just keep the TODO comment.